--- hicolor-icon-theme-0.12/Makefile.in.orig	2010-01-13 11:18:27.000000000 +0100
+++ hicolor-icon-theme-0.12/Makefile.in	2011-09-28 14:12:14.430478446 +0200
@@ -9,10 +9,10 @@
 distdir = $(PACKAGE)-$(VERSION)
 FILES = README Makefile.in configure.ac configure install-sh index.theme ChangeLog COPYING
 
-ICON_SIZES = 16x16 22x22 24x24 32x32 36x36 48x48 64x64 72x72 96x96 \
-  128x128 192x192 256x256 scalable
+ICON_SIZES = 8x8 16x16 20x20 22x22 24x24 32x32 36x36 42x42 48x48 64x64 72x72 96x96 \
+  128x128 160x160 192x192 256x256 512x512 scalable
 ICON_CONTEXTS = actions animations apps categories devices emblems emotes filesystems intl mimetypes places status stock
-STOCK_SUBS = chart code data form image io media navigation net object table text
+STOCK_SUBS = chart code data emoticons form image io media navigation net object table text
 
 all:
 	@:
